cita-cli/src/interactive.rs

```diff
@@ -424,3 +424,19 @@ impl GlobalConfig {
         println!("{}", output);
     }
 }
+
+fn remove_private(private_key: &str, blake_2b: bool) -> String {
+    if blake_2b {
+        private_key
+            .split_whitespace()
+            .filter(|key| !(key.starts_with("0x") && key.len() == 128 + 2))
+            .collect::<Vec<&str>>()
+            .join(" ")
+    } else {
+        private_key
+            .split_whitespace()
+            .filter(|key| !(key.starts_with("0x") && key.len() == 64 + 2))
+            .collect::<Vec<&str>>()
+            .join(" ")
```

Comment:
The difference is just `64` and `128` why write the main logic twice?

Reply:
Just copy .... emmm, I will change it immediately